docs/api/server/Environment-Variables.md

```diff
@@ -714,6 +725,90 @@ ONE_SERVICE_WORKER=true
 ONE_SERVICE_WORKER=false
 ```
 
+## `OTEL_EXPORTER_OTLP_LOGS_ENDPOINT`
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+When set, One App will emit OpenTelemetry logs over GRPC to the configured endpoint.
+
+**Shape**
+```bash
+OTEL_EXPORTER_OTLP_LOGS_ENDPOINT=String
+```
+
+**Example**
+```bash
+OTEL_EXPORTER_OTLP_LOGS_ENDPOINT=http://localhost:4318/v1/logs
+```
+
+## `OTEL_SERVICE_NAME`
+
+> ⚠️ Requires [`OTEL_EXPORTER_OTLP_LOGS_ENDPOINT`] to be set.
+
+**Runs In**
+* ✅ Production
+* ✅ Development
+
+Service name for OpenTelemtry resource.
+See [OTel Environment Variable Specification] for more details
+
+**Shape**
+```bash
+OTEL_SERVICE_NAME=String
+```
+
+**Example**
+```bash
+OTEL_SERVICE_NAME=MyApplication
+```
+
+**Default Value**
+```bash
+OTEL_SERVICE_NAME="One App"
```

Comment:
Can this value be set with quotes? If not, this default value can be misleading

Reply:
Yes